src/routes/propertiesRoute.ts

```diff
@@ -0,0 +1,131 @@
+    
+            const response = await dynamo.putItem({
+                    TableName: 'properties',
+                    Item: { 
+                        id: { S: newId },
+                        name: {S: body.name || ''},
+                        description : {S: body.description || ''},
+                        cover_image : {S: body.cover_image || ''}
+                    }
+            }).promise();
+    
+            return {
+                statusCode: 200,
+                body: JSON.stringify(response),
+            };
+
+        } catch(e){
+            return {
+                statusCode: 500,
+                body: JSON.stringify(e)
+            }
+        }
+    }
+}
+
+export function propertyUpdate() {
+    const dynamo = createDynamo();
+
+    return async (event: awsx.apigateway.Request) => {
+        try{
+            const id = event.pathParameters ? event.pathParameters.id : '';
+            const body = JSON.parse(event.body || '{}')
+    
+            const search = await dynamo.getItem({
+                TableName: 'properties',
+                Key: { "id": { "S": id.toString() } }
+            }).promise();
+
+            if (!search.Item) {
+                return {
+                    statusCode: 404,
+                    body: 'Item not found'
+                };
+            } else {
+                const response = await dynamo.putItem({
+                    TableName: 'properties',
+                    Item: { 
+                        id: { S: id },
+                        name: {S: body.name || search.Item.name},
+                        description : {S: body.description || search.Item.description},
+                        cover_image : {S: body.cover_image || search.Item.cover_image}
+                    }
+                }).promise();
+
+                return {
+                    statusCode: 200,
+                    body: JSON.stringify(response),
+                };
+            }
+
+        } catch(e){
+            return {
+                statusCode: 500,
+                body: JSON.stringify(e)
+            }
+        }
+    }
+}
+
+export function propertyGetById() {
+    const dynamo = createDynamo()
+
+    return async (event: awsx.apigateway.Request) => {
+        try{
+            const id = event.pathParameters ? event.pathParameters.id : '';
+            
+            const response = await dynamo.getItem({
+                TableName: 'properties',
+                Key: { "id": { "S": id.toString() } }
+            }).promise();
+    
+            return response.Item ? {
+                statusCode: 200,
+                body: JSON.stringify(response.Item),
+            } : {
+                statusCode: 404,
+                body: 'Not Found'
+            }          
+    
+        } catch(e){
+            return {
+                statusCode: 500,
+                body: JSON.stringify(e)
+            }
+        }
+    }
+}
+
+export function propertiesGet() {
```

Comment:
Please remove this api, it should be implemented with paging functionality. It is also important to avoid dynamodb scans as they will scan the whole table which is not desirable